src/debug.rs

```diff
@@ -52,6 +52,33 @@ pub fn dump_memory(start_address: *const usize, count: isize) {
     }
 }
 
+/// Use the LowLevelDebug capsule (if present) to indicate the given status
+/// code. If the capsule is not present, this is a no-op.
+#[inline(always)] // Improve reliability for relocation issues
+pub fn low_level_status_code(code: usize) {
+    unsafe {
+        crate::syscalls::command1_insecure(0x8, 1, code);
```

Comment:
Instead of hard-coding the constants here (which isn't very readable), could you define a `const DRIVER_NUMBER` and `mod command_nr` as is the practice for other drivers in libtock-rs?

Reply:
Done. There wasn't a good place to put the constant so I extracted the low level debug functionality into a new submodule of debug.